e2e/e2e_trans_test.go

```diff
@@ -20,16 +20,20 @@ import (
 func TestTransProcess(t *testing.T) {
 	//----- Recover account ---------
 	mnemonic := "test mnemonic"
+	baeUrl := "dex-api.fdgahl.cn"
```

Comment:
would this expose our entity in domain certification?

Reply:
hide it now